css/app.css

```diff
@@ -1,3 +1,820 @@
+/* reset&global */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+}
+
 body {
-  background-color: blue;
+  background-color: #f5f5f5;
+  max-width: 1920px;
+  margin: 0 auto;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+section {
+  margin: 5.3125rem 0;
+}
+
+.section-title {
+  margin-bottom: 1.25rem;
+}
+
+.section-title a {
+  color: #f42;
+}
+
+/* navbar-top */
+nav {
+  display: flex;
+}
+
+.nav-logo {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  background: #f42;
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  padding: 0 1.25rem;
+}
+
+.nav-logo img {
+  height: 1rem;
+  width: 4.125rem;
+}
+
+.navbar-container {
+  flex-grow: 1;
+}
+
+nav .nav-top {
+  height: 2rem;
+  background-color: black;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 2rem;
+  padding: 0 0.9375rem;
+}
+
+.nav-top-left,
+.nav-top-right {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.nav-main-right a {
+  color: rgb(190, 190, 190);
+  padding: 0.875rem 1.25rem;
+  border-left: 1px solid rgba(190, 190, 190, 0.5);
+  transform: rotateY(180deg);
+}
+
+.latest-funding-top-right a {
+  padding: 0 0.625rem;
+  color: #f42;
+  color: inherit;
+}
+
+.latest-funding-footer a {
+  color: #555;
+}
+
+.main-article-social,
+.main-article-social a {
+  font-size: 0.75rem;
+  color: #fff9;
+}
+
+.main-article-subarticles a {
+  font-size: 0.8125rem;
+  color: rgb(99, 99, 99);
+}
+
+.latest-deals-header > a {
+  color: #f42;
+}
+
+.nav-top-left li .active {
+  color: white;
+}
+
+/* navbar-main */
+nav .nav-main {
+  height: 2.8125rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 0.9375rem;
+  background-color: #fff;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.align-items-center{
    align-items: center;
}
```
**Html file**

```
<div class="align-items-center">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_